GOVERNANCE.md

```diff
@@ -0,0 +1,195 @@
+# Governance
+
+This document outlines the governance model for Biome. This document includes the contributor model, code review, merging, and the consequences and process for Code of Conduct violations.
+
+## Member Roles
+
+All members must follow the [Code of Conduct](CODE_OF_CONDUCT.md). Consequences for member violations are detailed in [Moderation](#moderation).
+
+### Lead
+
+Leads are the owners of the organisation.
+
+Leads have additional privileges over core contributors. Leads control and maintain sensitive project assets and act as tiebreakers in the event of disagreements.
+
+These assets and responsibilities are but are not limited to:
+- Access to social accounts
+- Administration privileges of the [Biome Discord server][discord]
+- Administration privileges of the [Biome GitHub org][gh-org]
+- Website accesses (hosting platform, domain name, etc.)
+- Ability to vote for new leads
+- Onboard new [core contributors](#core-contributor) and new leads;
+- Takes part in [project decisions](#project-direction-and-planning)
+
+The ownership of assets is only sometimes evenly distributed among all the leads.
+
+### Core Contributor
+
+Core Contributors are outstanding [maintainers](#maintainer), ambassadors of Biome organisation and lead by example the community.
+
+- Push access to the [Biome GitHub org][gh-org], this includes all repositories
+- `Core contributor` status on the [Biome Discord server][discord]
+- Takes part in [project decisions](#project-direction-and-planning)
+- Ability to [vote](#maintainer-nomination) on new maintainers and [vote](#core-contributor-nomination) on new core contributors
+- Onboard new [maintainers](#maintainer)
+
+#### Core contributor nomination
+
+Core contributors may either be nominated by another, [Core contributor](#core-contributor), or [lead](#lead).
+
+When nominating a core contributor, the person has to create a new thread in the `#core-contributors` channel with a significant explanation of why the person should join as [core contributor](#core-contributor).
```

Comment:
Are there guidelines or criteria for what constitutes a "significant explanation" when nominating someone?

Reply:
I reworded the sentence. I made it more generic. Let me know if you still have doubts.